lib/utils/downloadTemplateFromRepo.js

```diff
@@ -46,23 +46,27 @@ function validateUrl(url, hostname, service, owner, repo) {
 }
 
 /**
+ * isGhe = is github entreprise url?
  * @param {Object} url
+ * @param {Boolean} isGhe
  * @returns {Object}
  */
-function parseGitHubURL(url) {
+function parseGitHubURL(url, isGhe) {
```

Comment:
Can't you inspect the url and if it's not the `github.com` domain you'd expect it to be a GitHub enterprise domain? This way there would be no need to pass this boolean into the function.

Reply:
Done, i removed the useless boolean by doing the check in the function.